macros/paths_to_conversion.sql

```diff
@@ -191,7 +191,7 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
   with paths as (
     
     select
-      {% if var('snowplow__conversion_stitching') %}
+      {% if var('snowplow__conversion_stitching', false) %}
```

Comment:
In the default__ macro above, you refer to the view_stitching variable in the paths cte, why not here as well?

Reply:
Oh, I overlooked that one, good catch!